book/src/prefabs/how_to_define_prefabs_aggregate.md

```diff
@@ -32,6 +32,9 @@ If you intend to include a [`Component`] that has not yet got a corresponding [`
     In these examples, `Named`, `Position`, and `Weapon` all derive [`PrefabData`].
 
     ```rust,edition2018,no_run,noplaypen
+    # extern crate amethyst;
```

Comment:
Do we need these? Everyone will be using 2018 now, right?

Reply:
Yes, we need them, since `mdbook` uses a regex to determine which crates are needed (and it only looks for `extern crate`).

This won't be visible to the reader, though.

Reply:
Cool, makes sense.